.circleci/config.yml

```diff
@@ -133,6 +133,16 @@ jobs:
     steps:
       *test_steps
 
+  ruby25rails60beta:
+    docker:
+    - image: circleci/ruby:2.5.1
+
+    environment:
+      BUNDLE_GEMFILE: test/gemfiles/Gemfile-Rails-6-0-beta
+
+    steps:
+      *test_steps
+
```

Comment:
I guess we can add the whole matrix and test against all supported rubies since we are at it?

Reply:
We need also add ruby 2.6

Reply:
@deivid-rodriguez according to the guideline Rails 6 requires ruby 2.5 or newer
https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#ruby-versions

Reply:
You're absolutely right, my bad! This PR is going in then!